Description: Fix SEGV when GLX not available.
 Revised in the light of vadz's comment:
 http://trac.wxwidgets.org/ticket/13566#comment:1
Bug-Debian: http://bugs.debian.org/575536
Forwarded: http://trac.wxwidgets.org/ticket/13566
Last-Update: 2011-10-18

diff --git a/src/gtk/glcanvas.cpp b/src/gtk/glcanvas.cpp
index 24ab43f..d30935c 100644
--- a/src/gtk/glcanvas.cpp
+++ b/src/gtk/glcanvas.cpp
@@ -327,6 +327,8 @@ bool wxGLCanvas::Create( wxWindow *parent,
             m_canFreeFBC = true;
         }
         m_fbc = fbc;  // save for later use
+        if (!m_fbc)
+            wxWindow::Create(parent, id, pos, size, style, name);
         wxCHECK_MSG( m_fbc, false, _T("required FBConfig couldn't be found") );
     }
 
@@ -350,6 +352,8 @@ bool wxGLCanvas::Create( wxWindow *parent,
 
     m_vi = vi;  // save for later use
 
+    if (!m_vi)
+        wxWindow::Create(parent, id, pos, size, style, name);
     wxCHECK_MSG( m_vi, false, _T("required visual couldn't be found") );
 
     // watch for the "parent-set" signal on m_wxwindow so we can set colormap
